packages/salesforcedx-vscode-core/src/util/rootWorkspace.ts

```diff
@@ -0,0 +1,17 @@
+import { workspace, WorkspaceFolder } from 'vscode';
+
+export function hasRootWorkspace(ws: typeof workspace = workspace ) {
+  return (ws && ws.workspaceFolders && ws.workspaceFolders.length > 0);
+}
+
+export function getRootWorkspace(): WorkspaceFolder {
+  return hasRootWorkspace() ? workspace.workspaceFolders![0] : {} as WorkspaceFolder;
+}
+
+export function getRootWorkspaceFsPath(): string {
+  return getRootWorkspace().uri ? getRootWorkspace().uri.fsPath : '';
+}
+
+export function getRootWorkspacePath(): string {
```

Comment:
I think we should remove this getter in order to prevent usage of `uri.path`.

Reply:
OK, I ended up removing the other getter ```getRootWorkspaceFsPath()```, and kept ```getRootWorkspacePath()```, but had it dig out the uri.fsPath property and return that.   

I guess to me it seemed to express the user's intent without unnecessary specificity of the underlying model.